### PR TITLE
Add aggregate ratings to comments

### PR DESCRIPTION
**Pull request by shinnlok**
Fixes: #438

To be honest, I'm not sure about any of this. If I'm far off, I may have to unassign myself. :sweat_smile:

### CODE REVIEW

**Review by baltpeter — DISMISSED** [review]
Thanks for the PR, @shinnlok! That's pretty much the implementation I had in mind. :) 

I would like to have a nicer styling, though. I think we should definitely move the CommentSummary to the top. Maybe you could even think of a nicer way of styling it. But otherwise, we can just merge and leave that as another issue.

**Comment by shinnlok** [comment]
@baltpeter Here's my attempt. I'm a backend guy so.. feel free to ask me to revert the styling. :)

Screen Shot 2020-10-15 at 12 02 23 PM

File: layouts/company/single.html [hunk 1]
```diff
@@ -136,11 +136,15 @@ <h2>{{ T "cdb-related-companies" }}</h2>
...
+
+{{ .Scratch.Add "ldJsonId" .Permalink }}
+{{ .Scratch.Add "ldJsonId" "#company" }}
+
 <script type="application/ld+json">
 {
   "@context": "http://schema.org",
   "@type": "Organization",
-  "@id": "{{ .Params.slug }}",
+  "@id": {{ .Scratch.Get "ldJsonId" }},
```
Comment (shinnlok) [inline layouts/company/single.html:147]: I couldn't figure out how to tell Hugo not to escape.

**Comment by baltpeter** [comment]
Thanks, I do like the idea! I'll take over the PR and try to refine it a little. As it may take a bit for me to have the time for that, though, I'll already mark the PR as `hacktoberfest-accepted`. :)

**Comment by baltpeter** [comment]
Here's my go at this (finally :|). I think I prefer the more minimal style.

Since I've now made quite a few changes, would either of you please do the review, @zner0L @mal-tee?

**Review by zner0L — APPROVED** [review]
Looks good to me

**Pull request merged**